lib/fragment.js

```diff
@@ -9,6 +9,13 @@ const zlib = require('zlib');
 const { globalTracer, Tags } = require('opentracing');
 const tracer = globalTracer();
 
+const getValue = value => {
```

Comment:
this function seems misnamed. It's not getting the value, it's doing a kind of extended truthiness-check, isn't it? Maybe better named `hasValue`?